src/js/components/InfiniteScroll/stories/Replace.js

```diff
@@ -22,7 +22,12 @@ const InfiniteScrollReplace = props => (
 );
 
 if (!isChromatic()) {
-  storiesOf('InfiniteScroll', module).add('Replace', () => (
-    <InfiniteScrollReplace replace />
-  ));
+  storiesOf('InfiniteScroll', module)
+    .add('Replace', () => <InfiniteScrollReplace replace />)
+    .add('Replace with Show 28th item', () => (
+      <InfiniteScrollReplace replace show={27} />
+    ))
+    .add('Replace with Show 88th item', () => (
+      <InfiniteScrollReplace replace show={87} />
+    ));
```

Comment:
Why do we need two stories that point to a specific item with `replace`? I'm asking because we want to keep the stories concise.

Reply:
Since we have limited tools to test behaviors, Storybook is our best current option. I am fine removing the stories once this work is merged in or even relocating them, but they are valuable for QA effort.

In this case in particular, behaviors are different when show > step vs. show < step. We need both circumstances to work. When we only tested for one variant, we got false positives and scenarios where the component did not function properly.

Again happy to remove once we feel good about all the fixes on this component.

Reply:
I agree we should keep it, but maybe adding a comment on the stories regarding show > step vs. show < step, as a user at a first glance it might not be obvious the difference between the two stories.

Reply:
I could put both examples into one story, each labeled with the scenario which would demonstrate the functionality. That way we could reduce the clutter, but maintain QA scenarios.

The monorepo work may also be an opportunity to provide a place for these sorts of QA items, while leaving Storybook clean for users and documentation.

Reply:
>I could put both examples into one story, each labeled with the scenario which would demonstrate the functionality. That way we could reduce the clutter, but maintain QA scenarios.

I think I like that.